python/grass/gunittest/testsuite/test_module_assertions.py

```diff
@@ -30,18 +31,28 @@ def setUp(self):
 
     def test_runModule(self):
         """Correct and incorrect Module used in runModule()"""
+        """Run PyGRASS module. Runs the module and raises an exception if the module ends with non-zero return code. 
+        Usually, this is the same as testing the return code and raising exception but by using this method, you give testing framework more control over the execution, error handling and storing of output.
+        In terms of testing framework, this function causes a common error, not a test failure."""
```

Comment:
The comment is a copy-paste of documentation of the runModule method. No reason to repeat it here every time the method is called.

https://grass.osgeo.org/grass80/manuals/libpython/gunittest.html#gunittest.case.TestCase.assertModule

This would be a place to describe behavior of the test code.